### DISCUSSION TITLE
A security issue downloading Visual Studio and installing the programs. Python virus or something.

### DISCUSSION THREAD

**Discussion by Terry3373** [community/community#64073] [New to GitHub]
![Screenshot 2023-08-17 052404](https://github.com/community/community/assets/86999804/92e9f7c6-3319-43d9-b005-df2f08bd7b94)

**Comment by bronglil** [comment]
The error message "Security issue found" means that there is a problem with the way Visual Studio was downloaded or installed. This could be due to a number of factors, such as:

Downloading Visual Studio from an untrusted source.
Installing Visual Studio over an existing installation.
Corrupted installation files.
Antivirus software blocking the installation.
If you are getting this error message, you should first try to download Visual Studio from the official Microsoft website. You should also make sure that your antivirus software is up to date and that it is not blocking the installation. If you are still getting the error message, you can try the following steps:

Uninstall Visual Studio completely.
Restart your computer.
Download and install Visual Studio again.
Run a virus scan of your computer.
If you are still having problems, you can contact Microsoft support for help.

As for the Python virus, there is no evidence that Visual Studio is infected with a Python virus. However, it is always a good idea to run a virus scan of your computer after downloading and installing any new software.

Here are some additional things you can do to protect your computer from security issues:

Keep your operating system and software up to date.
Use a strong password and do not share it with anyone.
Be careful about what websites you visit and what files you open.
Install antivirus and anti-malware software and keep it up to date.
Be aware of the latest security threats and how to protect yourself from them.
I hope this helps!

**Reply by Terry3373** [reply]
Check it out. I was able to actually send you the. encryption they were trying to use and. the actual weird codeing

**Reply by Terry3373** [reply]
Didn't Microsoft discontinue using python altogether? Was that for security reasons? Or is that even valid?

**Reply by Terry3373** [reply]
I'm running Windows 11 pro it's beautiful.

**Reply by bronglil** [reply]
Make sure you're using the latest version of Python. Outdated software can have security vulnerabilities that malware can exploit. Upgrade to the latest version from the official Python website or try Reset Python Environment.

**Comment by Terry3373** [comment]
![image](https://github.com/community/community/assets/86999804/2d643332-37ef-4024-8f7a-ac33606f7630)

**Reply by Terry3373** [reply]
![image](https://github.com/community/community/assets/86999804/3c88777d-dd71-4570-9004-9804e16346cf)
